packages/core/package.json

```
@@ -714,21 +714,28 @@
                 {
                     "id": "aws.amazonq.codewhisperer",
                     "name": "%AWS.amazonq.codewhisperer.title%",
-                    "when": "!isCloud9 && !aws.isSageMaker"
+                    "when": "!isCloud9 && !aws.isSageMaker && !aws.explorer.showAuthView"
                 },
                 {
                     "id": "aws.explorer",
                     "name": "%AWS.lambda.explorerTitle%",
-                    "when": "isCloud9 || !aws.isWebExtHost"
+                    "when": "(isCloud9 || !aws.isWebExtHost) && !aws.explorer.showAuthView"
                 },
                 {
                     "id": "aws.cdk",
-                    "name": "%AWS.cdk.explorerTitle%"
+                    "name": "%AWS.cdk.explorerTitle%",
+                    "when": "!aws.explorer.showAuthView"
                 },
                 {
                     "id": "aws.codecatalyst",
                     "name": "%AWS.codecatalyst.explorerTitle%",
-                    "when": "!isCloud9 && !aws.isSageMaker || isCloud9CodeCatalyst"
+                    "when": "(!isCloud9 && !aws.isSageMaker || isCloud9CodeCatalyst) && !aws.explorer.showAuthView"
+                },
+                {
+                    "type": "webview",
+                    "id": "aws.AmazonCommonAuth",
```

Please use naming conventions that are consistent with the project. 

```suggestion
                    "id": "aws.amazonCommonAuth",
```